python/adelphi/tests/util/schemadiff.py

```diff
@@ -0,0 +1,27 @@
+import base64
+import hashlib
+import logging
+
+# Imports required by the __main__ case below
+import sys
+
+def cqlDigestGenerator(schemaFile):
+    buff = ""
+    for line in open(schemaFile):
+        realLine = line.strip()
+        if len(realLine) == 0 or realLine.isspace():
+            continue
+        buff += (" " if len(buff) > 0 else "")
+        buff += realLine
+        if realLine.endswith(';'):
+            cqlStmt = buff
+            buff = ""
+            m = hashlib.sha256()
```

Comment:
Fun fact I learned while working on this code: there's no way to reset a Python digest object (like the sha256 object we have here).  You just feed it data and when you call digest() it gives you the digest of whatever data it's seen thus far.  So if the schema we get from adelphi's "export-cql" is in a different order from what's in the reference schema you'll get digest mismatches for the same content because of the change in ordering.

And we know there are differences in ordering between those two files... that was one of the prime movers that led to checking things in Python via digests in the first place.

Creating a new digest for each complete statement and just using it immediately addressed the problem.  It strikes me as somewhat wasteful but it seems to be the Pythonic way for handling something like this.